### PR TITLE
Functionality improvements

### LINKED ISSUE #100: Something wrong in /F20/
`\todo{something is wrong, here}`

Was at /F20/, i removed it cause it disturbed the build process.

### PR DESCRIPTION
**Pull request by jGleitz**
An attempt to streamline chapter “Functionality”.

Fixes the chapter’s numbering.
Contains multiple wording fixes. Removes the explicit statement of what will be saved to the PCM, as this is covered by /F200/ (“all its results”). Adds an explicit reference to StoEx.

fixes #100

### CODE REVIEW

**Comment by jGleitz** [comment]
I will re-introduce “Beagle ensures that the source files were not changed when the user resumes
- the measurement.” in my next PR targeting #127.

**Comment by jGleitz** [comment]
[View this PR’s rendered PDF](http://jgleitz.github.io/Beagle/branches/functionality-fixes/Requirements%20Specification.pdf)

**Comment by acBerger** [comment]
lgtm

File: doc/Requirements Specification/Parts/Functional Requirements.lyx [hunk 1]
```diff
@@ -525,40 +496,25 @@ name "/OF90/"
...
-
-
-\backslash
-gls{component}
-\end_layout
-
-\end_inset
-
-.
- Beagle ensures that the source files were not changed when the user resumes
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Functional Requirements.lyx]: Why do you removed this?

Comment (jGleitz) [inline doc/Requirements Specification/Parts/Functional Requirements.lyx]: Please see my comment above

> I will re-introduce “Beagle ensures that the source files were not changed when the user resumes the measurement.” in my next PR targeting #127.

Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Functional Requirements.lyx]: Allright, sorry.

**Comment by RomanLangrehr** [comment]
Lgtm

**Pull request merged**